drools-wb-webapp/Readme.md

```diff
@@ -10,6 +10,6 @@ The jars included as additional sources should be set as **provided** in the **d
 
 NOTE
 ----
-Be sure you increased your inotify limit before start the project. See [here](https://github.com/kiegroup/droolsjbpm-build-bootstrap/blob/master/README.md#linux-inotify) for more detail".
+Be sure you increased your inotify limit before start the project. See [here](https://github.com/kiegroup/kie-wb-common/tree/master/kie-wb-common-stunner#ide-environment-setup) for more detail".
```

Comment:
please remove the trailing quotation mark - it doesn't seem to have any opening counterpart.

Reply:
done